### ISSUE TITLE
StackSet lambdas are zipped and correctrly referenced but not uploaded during deployment

### ISSUE THREAD

**Issue by mguja-fcc** [aws-solutions/quota-monitor-for-aws#187]
**Describe the bug**
StackSet QM-TA-Spoke-StackSet and QM-SQ-Spoke-StackSet lambdas are zipped in cdk.out folder and s3 location correctly referred in the template but actual zip files are not uploaded into s3 buucket. 

**To Reproduce**

1. run npm run orgHub:deploy -- deploy quota-monitor-hub to deploy the solution
2. Navigate to StackSet, for example, QM-TA-Spoke-StackSet and click add stack to StackSet
3. Stack deployment fails due to error s3 kay was not found

**Expected behavior**
StackSet stack is deployed.

**Please complete the following information about the solution:**

- [ ] Version: [e.g. v6.2.5]

**Comment by shujacks** [comment]
Thanks for reaching out, we will look into this and get back with you soon.

**Comment by mguja-fcc** [comment]
Any update on this ? we want to deploy though code without a need of manual steps. Thanks.

**Comment by G-Lenz** [comment]
Just to make sure I'm understanding correctly, the stack sets won't deploy due to an s3 key error but the hub stack was deployed successfully, is that correct?

**Comment by mguja-fcc** [comment]
Stackset instances won't deploy for both QM-TA-Spoke-StackSet and QM-SQ-Spoke-StackSet

**Comment by abewub** [comment]
Yes `npm run orgHub:deploy` deploys only the hub stack. We will add a backlog item for fixing the template urls to the stacksets.

**Comment by sanjay-reddy-kandi** [comment]
Fixed in `v6.3.0`. Closing issue.
